packages/config/config/devices/0x0346/keypad.json

```diff
@@ -0,0 +1,387 @@
+// Ring Keypad
+// Param information taken from https://support.ring.com/hc/en-us/article_attachments/360051851012/Keypad_Zwave_UK.pdf
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "Keypad",
+	"description": "Keypad",
+	"devices": [
+		{
+			"productType": "0x0101",
+			"productId": "0x0201"
+		},
+		{
+			"productType": "0x0101",
+			"productId": "0x0202"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Interval",
+			"unit": "min",
+			"valueSize": 2,
+			"minValue": 70,
+			"maxValue": 1440,
+			"defaultValue": 70,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Z-Wave Sleep Timeout",
+			"unit": "sec",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Co-Proc Is Alive",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "No",
+					"value": 0
+				},
+				{
+					"label": "Yes",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Proximity Timeout",
+			"unit": "sec",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 30,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"label": "Button Press Timeout",
+			"unit": "sec",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 30,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "Status Change Timeout",
+			"unit": "sec",
+			"valueSize": 1,
+			"minValue": 10,
+			"maxValue": 60,
+			"defaultValue": 15,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"7": {
+			"label": "Power Mode",
```

Comment:
I'd call this Enable / Disable Extended Battery Life Mode. And make the options Enable and Disable. This matters because we're templating devices and the more consistency we can have the better.